app/services/api/webflow-api.ts

```diff
@@ -31,75 +31,137 @@ import {
   cleanedWorkshops,
   convertScheduleToScheduleCard,
 } from "./webflow-helpers"
+import { queryClient } from "./react-query"
 
-const useWebflowAPI = <T>(key: string, collectionId: string, enabled = true) =>
-  useQuery({
-    queryKey: [key],
-    queryFn: async () => {
-      const { data } = await axiosInstance.get<PaginatedItems<T>>(
-        `/collections/${collectionId}/items`,
-      )
-      return data.items
-    },
-    enabled,
-  })
```

Comment:
Decided to replace this with having the `useQuery` options directly accessible, so that we can compose them later with hooks like `useQueries` or prefetching specific queries using their `queryKey` and `queryFn`.

Reply:
it looks like they're the same functions just renamed differently. maybe we can still use this reusable component but expose more of the items you're looking for in the function?

Reply:
The main difference here is that we can directly pass the `queryKey` and `queryFn` instead of accessing it through the `useWebflowAPI` wrapper.

This isn't a huge difference, except exposing the `queryKey` and `queryFn` reference lets use them later in other places like `useQueries` for prefetching multiple queries. Which we can't do if they are wrapped in `useWebflowAPI`.

I thought about exposing `queryKey` and `queryFn` as a part of the existing hooks, but the problem with that is `useQuery` will attempt to fetch using the `queryFn` on mount, which is not ideal for controlling prefetching. That being said,[ I did find a doc example for `enabled`,](https://tanstack.com/query/v4/docs/react/reference/useQuery) which may let us still use the hook but not fetch on mount.
<img width="552" alt="Screenshot 2023-04-12 at 10 30 56 AM" src="https://user-images.githubusercontent.com/37849890/231537587-540966ff-f0fd-44de-8878-8cdb1df67b6a.png">

Reply:
Here is the usage later that I'm referring to: https://github.com/infinitered/ChainReactApp2023/pull/108/files#diff-f8eeb82e0e0624e99d187ff5e43f122a24c4a517c0102ad6048e434098d6a843R121-R192